<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::West</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="usesScrollButtons">
         <bool>true</bool>
        </property>
        <property name="documentMode">
         <bool>false</bool>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Выбор Графика</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QTreeView" name="treeView">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <attribute name="title">
          <string>Настройка Графика</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="26" column="0">
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Добавить график</string>
              </property>
             </widget>
            </item>
            <item row="12" column="0">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <weight>50</weight>
                <bold>false</bold>
                <underline>false</underline>
                <stylestrategy>PreferDefault</stylestrategy>
               </font>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="text">
               <string>Осреднение:</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="openExternalLinks">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="18" column="0">
             <widget class="Line" name="line_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="17" column="0">
             <widget class="QComboBox" name="comboBox_2">
              <item>
               <property name="text">
                <string>По умолчанию</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Синий</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Красный</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Зелёный</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Фиолетовый</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Чёрный</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="28" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="25" column="0">
             <widget class="Line" name="line">
              <property name="inputMethodHints">
               <set>Qt::ImhNone</set>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="14" column="0">
             <widget class="QComboBox" name="comboBox">
              <item>
               <property name="text">
                <string>Нет</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>За час</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>За каждые 3 часа</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>За сутки</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="19" column="0">
             <widget class="QCheckBox" name="checkBox_date">
              <property name="text">
               <string>Ограничить по дате</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="Line" name="line_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>Удалить артефакты</string>
              </property>
             </widget>
            </item>
            <item row="21" column="0">
             <layout class="QFormLayout" name="formLayout_6">
              <property name="topMargin">
               <number>0</number>
              </property>
              <item row="0" column="1">
               <widget class="QDateTimeEdit" name="dateTimeEdit_2">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_4">
                <property name="minimumSize">
                 <size>
                  <width>40</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Начало:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_3">
                <property name="minimumSize">
                 <size>
                  <width>40</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Конец:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDateTimeEdit" name="dateTimeEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="checkBox_name">
              <property name="text">
               <string>Указать имя графика</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="lineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="tabletTracking">
               <bool>true</bool>
              </property>
              <property name="focusPolicy">
               <enum>Qt::StrongFocus</enum>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::NoContextMenu</enum>
              </property>
              <property name="inputMethodHints">
               <set>Qt::ImhNone</set>
              </property>
              <property name="inputMask">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>По умолчанию</string>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Normal</enum>
              </property>
              <property name="dragEnabled">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="cursorMoveStyle">
               <enum>Qt::VisualMoveStyle</enum>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="Line" name="line_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Тип графика:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QComboBox" name="comboBox_3">
              <item>
               <property name="text">
                <string>Линейный</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Сплайн</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="15" column="0">
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="16" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Цвет:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Общие Настройки</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QCheckBox" name="checkBox_edit">
              <property name="text">
               <string>Своя настройка</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Ось Y:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Имя:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_3">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::StrongFocus</enum>
                </property>
                <property name="text">
                 <string>По умолчанию</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout">
              <property name="topMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Минимум:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="spinBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimum">
                 <number>-1000000</number>
                </property>
                <property name="maximum">
                 <number>1000000</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Максимум:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="spinBox_2">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimum">
                 <number>-1000000</number>
                </property>
                <property name="maximum">
                 <number>1000000</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Ось X:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_9">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Имя:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_2">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::StrongFocus</enum>
                </property>
                <property name="text">
                 <string>По умолчанию</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_2">
              <property name="topMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_13">
                <property name="minimumSize">
                 <size>
                  <width>41</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Начало:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDateTimeEdit" name="dateTimeEdit_3">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Конец:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDateTimeEdit" name="dateTimeEdit_4">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QChartView" name="graphicsView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen_CSV"/>
    <addaction name="action_JSON"/>
    <addaction name="separator"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>График</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionOpen_CSV">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Открыть CSV</string>
   </property>
   <property name="iconText">
    <string>Открыть CSV</string>
   </property>
  </action>
  <action name="action_JSON">
   <property name="text">
    <string>Открыть JSON</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Добавить график из интернета</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Удалить все графики с раб. поля</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
